src/main/java/com/vdurmont/emoji/EmojiParser.java

```diff
@@ -84,6 +84,23 @@ public String transform(UnicodeCandidate unicodeCandidate) {
     return parseFromUnicode(input, emojiTransformer);
   }
 
+  /**
+   * Replace all emojis with character
+   *
+   * @param str the string to process
+   * @param chr the character to be replaced
```

Comment:
I would rename this argument because it can be anything, not just a character.
Something like:
```
@param replacement the string that will replace all the emojis
```

Reply:
Ok I'll do soon